client/state/sites/actions.js

```diff
@@ -14,19 +22,18 @@ import {
 	SITES_REQUEST_FAILURE,
 	SITES_UPDATE
 } from 'state/action-types';
-import { omit } from 'lodash';
 
 /**
  * Returns an action object to be used in signalling that a site has been
  * deleted.
  *
- * @param  {Object} site Site received
+ * @param  {Number} siteId Site id received
```

Comment:
Actually, as far as I can see right now, they're still not aligned. Compare lines 30 and 31 in client/state/sites/actions.js and you'll see what I mean:

```
 * @param  {Number} siteId ID of deleted site
 * @return {Object}      Action object
```